src/components/Features/TimerButton.tsx

```diff
@@ -1,11 +1,18 @@
-const TimerButton = ({ isRunning, toggleTimer }) => {
+const ButtonClassName = (isRunning: boolean): string => {
```

Comment:
- It is a function so you should change naming convention - e.g. getButtonColor;
- no need for bracers and return statement - use oneliner function